backends/xnnpack/_passes/channels_last_tagged_reshape_pass.py

```diff
@@ -170,15 +190,19 @@ def insert_dq_copy_q(
                 target=exir_ops.edge.quantized_decomposed.dequantize_per_tensor.default,
                 args=(before,) + q_params,
             )
-            dq.meta = before.meta
+            dq.meta = before.meta.copy()
 
         with graph_module.graph.inserting_after(copy):
             q = self.create_call_function_node(
                 graph_module=graph_module,
                 target=exir_ops.edge.quantized_decomposed.quantize_per_tensor.default,
                 args=(copy,) + q_params,
             )
-            q.meta = copy.meta
+            q.meta = copy.meta.copy()
+
+            # Always tag q/dq as implicit
+            tag_as_implicit_q_dq(dq)
```

Comment:
I was talking with @mcr229 about this, this can be brittle i.e. implicit tagging for q/dq in >1 places. Do we want to add a pass in the future to verify if the tagging is correct or not. Just for validation? Feel free to create an issue, don't have to add it right away.